mono/metadata/icall-def-netcore.h

```diff
@@ -48,6 +47,7 @@ HANDLES(ARRAY_7, "GetLowerBound",    ves_icall_System_Array_GetLowerBound, gint3
 HANDLES(ARRAY_8, "GetRank",          ves_icall_System_Array_GetRank, gint32, 1, (MonoObject))
 HANDLES(ARRAY_9, "GetValue",         ves_icall_System_Array_GetValue, MonoObject, 2, (MonoArray, MonoArray))
 HANDLES(ARRAY_10, "GetValueImpl",    ves_icall_System_Array_GetValueImpl, MonoObject, 2, (MonoArray, guint32))
+HANDLES(ARRAY_10a, "InternalCreate",   ves_icall_System_Array_InternalCreate, MonoArray, 4, (MonoReflectionType, gint32, gint32_ptr, gint32_ptr))
```

Comment:
Suggest use pointer to pointer instead of handles.

Reply:
@jaykrell um.. what do you mean? pass Type via pointer?

Reply:
Most instances of MonoReflectionType in icalls could be replaced by pointers (see is_subclass_of) without losing safety. Not sure if that's what Jay meant.

Reply:
@filipnavara @jaykrell ok, converted to NOHANDLES